docs/user_guide/Hiera_Overview.rst

```diff
@@ -73,58 +77,64 @@ site class ``/etc/puppetlabs/code/environments/simp/modules/site/manifests/my_si
 For example, to include tftpboot linux\_model and assign\_host on your
 puppet server, puppet.your.domain:
 
-Add the following code to a file tftpboot.pp in your site/manifests directory:
 
-.. code-block:: ruby
+SIMP File Structure
+===================
 
-        # in /etc/puppetlabs/code/environments/simp/modules/site/manifests/tftpboot.pp
-        # Set KSSERVER statically or use Hiera for lookup
+In the Puppet environment, which is located at
```

Comment:
The default puppet environment in SIMP, located at /etc/puppetlabs/code/environments/simp, contains almost...